src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -83,6 +83,8 @@ public string GitHubRepository
 
         public string GitHubBranch { get; set; }
 
+        public bool PublishUsingPipelines { get; set; }
```

Comment:
Where is this field used? I think we should include the code that will consume this value along with adding the value.

Reply:
This is going to be consumed by ReleasePipelineRunner. I need to make the Arcade build-def to pass this parameter to `PushMetadataToBuildAssetRegistry` before I can patch `ReleasePipelineRunner`, though. Otherwise it will assume Arcade isn't publishing using pipelines.